src/main/java/runge/worldcities/WorldCities.java

```diff
@@ -0,0 +1,60 @@
+package runge.worldcities;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class WorldCities
+{
+
+    private double cityLat;
+    private double cityLng;
+    private String closestCity;
+    private CSVParser parser;
+
+    private final HashMap<String, ArrayList<Double>> cities = new HashMap<>();
+
+    public WorldCities() throws IOException {
+        File csvData = new File("worldcities.csv");
+        parser = CSVParser.parse(csvData, Charset.defaultCharset(), CSVFormat.RFC4180);
+    }
+
+    public String getClosestCity(double givenLat, double givenLon)
+    {
+        double closestDistance = Double.MAX_VALUE;
+        double distance = Math.sqrt((givenLat - cityLat) * (givenLat - cityLat) * (givenLon - cityLng) * (givenLon - cityLng));
```

Comment:
What does this line do?

Reply:
31? that's the distance equation?

Reply:
Distance between what and what?

Reply:
the given lat/lng from url and the current city that I am looking at lat/lng

Reply:
Current city that you're looking at? Where does that happen?